src/Event/EventType/Wsj/EventTypeWsj.php

```diff
@@ -81,4 +81,28 @@ public function getLanguages(): array
             'cs' => '🇨🇿 Česky',
         ];
     }
+
+    /**
+     * @return array<string>
+     */
+    public function getPreferredPositionsOptions(): array
+    {
+        return array_merge(
+            parent::getPreferredPositionsOptions(),
```

Comment:
chceš to skutečně mergovat? Klidně si udělej vlastní seznam, až budete chtít něco z toho defaultního vyhodit, tak to oceníš